src/cljs/rems/actions/add_member.cljs

```diff
@@ -22,12 +22,23 @@
                ::selected-member nil)
     ::fetch-potential-members #(rf/dispatch [::set-potential-members %])}))
 
-(rf/reg-sub ::potential-members (fn [db _] (::potential-members db)))
+(defn potential-members [db]
+    (let [application (get-in db [:rems.application/application :data])
+          applicant (get-in application [:application/applicant])
+          members (get-in application [:application/members])
+          existing-ids (set (concat [(:userid applicant)]
+                                    (map :userid members)))]
+      (->> ::potential-members db
+           (remove (comp existing-ids :userid))
+           (map atoms/enrich-user))))
+
+(rf/reg-sub ::potential-members (fn [db _] (potential-members db)))
```

Comment:
We could remove this extraneous inline `fn` if the only thing is calling `potential-members`, or inline `potential-members` here. Something like this?
```
(rf/reg-sub ::potential-members potential-members)
```